es6-classes/circle.js

```diff
@@ -0,0 +1,18 @@
+/* eslint-disable no-undef -- Shape is imported by .html */
+/* exported Circle */
+class Circle extends Shape {
+  constructor(area, perimeter, radius) {
+    super(area, perimeter);
```

Comment:
Utilizing your `radius`, you need to calculate `area` & `perimeter`, and that's what you'll pass into `super`